rosbag2_storage_default_plugins/src/rosbag2_storage_default_plugins/sqlite/sqlite_storage.cpp

```diff
@@ -235,14 +221,14 @@ std::string SqliteStorage::get_storage_identifier() const
 
 std::string SqliteStorage::get_relative_path() const
 {
-  return database_name_;
+  return relative_path_;
```

Comment:
I thought we talked about this somewhere, but I think it makes sense to rename this function and variable name to `get_relative_file_path` to make clear that we talk about the file and not the directory or something else.

Reply:
Renamed `get_relative_path` to `get_relative_file_path`